pkg/api/admin/openshiftcluster.go

```diff
@@ -130,14 +130,23 @@ const (
 	MTU3900 MTUSize = 3900
 )
 
+// UserDefinedRouting represents
+type UserDefinedRouting string
+
+const (
+	UserDefinedRoutingEnabled  UserDefinedRouting = "Enabled"
+	UserDefinedRoutingDisabled UserDefinedRouting = "Disabled"
+)
+
 // NetworkProfile represents a network profile.
 type NetworkProfile struct {
 	// The software defined network (SDN) to use when installing the cluster.
 	SoftwareDefinedNetwork SoftwareDefinedNetwork `json:"softwareDefinedNetwork,omitempty"`
 
-	PodCIDR     string  `json:"podCidr,omitempty"`
-	ServiceCIDR string  `json:"serviceCidr,omitempty"`
-	MTUSize     MTUSize `json:"mtuSize,omitempty"`
+	PodCIDR            string             `json:"podCidr,omitempty"`
+	ServiceCIDR        string             `json:"serviceCidr,omitempty"`
+	MTUSize            MTUSize            `json:"mtuSize,omitempty"`
+	UserDefinedRouting UserDefinedRouting `json:"userDefinedRouting,omitempty"`
```

Comment:
After some discussion with Jerome, we decided we want to do something like this instead.  This will allow us to support more outbound types in the future if desired.  I'm not sure what to call the "default".  I believe within OCP documentation it's `LoadbalancerOutboundType` so maybe that's a good thing to set it as instead of `Default`.

```
OutboundType OutboundType
```

```const (
OutboundTypeUserDefinedRouting OutboundType = "UserDefinedRouting"
OutboundTypeDefault OutboundType = "Default"
```

Reply:
+1